metabrainz/static/css/theme/boostrap/list-group.less

```diff
@@ -21,7 +21,7 @@
 .list-group-item {
   position: relative;
   display: block;
-  padding: 10px 15px;
+  padding: 10px 7.5px;
```

Comment:
Hey, I think editing this file doesn't effect the navbar as it doesn't inherit any styles from this class. Can you revert this back ?